content/applications/hr/attendances/kiosks.rst

```diff
@@ -0,0 +1,187 @@
+======
+Kiosks
+======
+
+Odoo's *Attendances* app allows employees to check in and out of work directly from the database, or
+from a kiosk.
+
+Only employees with access to the database can check in and out from the database, and they are
+referred to as *users*. Kiosks are needed for employees who do **not** have access to the database.
+
+A kiosk is a dedicated device (a PC, tablet, or mobile phone) for employees to use when they check
+in and out.
```

Comment:
Suggestion to make a small adjustment to the order of instruction here, to introduce a kiosk and then add the layers of detail for the conditions of users vs external users :)

Also link to the _dedicated devices_
```suggestion
A kiosk is a :doc:`dedicated device <hardware>` (a PC, tablet, or mobile phone) for employees to use
when they check in and out.

Kiosks are needed for employees who do **not** have access to the database.

Only employees with access to the database can check in and out from the *Attendances* app, and they
are referred to as *users*.
```